ansible/roles/ml-analytics-service/defaults/main.yml

```diff
@@ -7,11 +7,11 @@ config_path: "/opt/sparkjobs/ml-analytics-service"
 SYSTEMDDIR: "/lib/systemd/system"
 ml_analytics_version: "release-4.1.0"
 ml_survey_cloud_storage: "AZURE"
-ml_analytics_survey_app_name: "{{ ml_cloud_config.CLOUD_STORAGE }}"
+ml_analytics_survey_app_name: "sunbirdsurvey"
 ml_analytics_integrated_app_name: "sunbird"
 ml_analytics_survey_service: "http://{{private_ingressgateway_ip}}/ml-survey-service"
 ml_analytics_evidence_base_url: "https://{{ml_analytics_azure_account_name}}.blob.core.windows.net/{{ml_analytics_azure_container_name}}/"
-ml_analytics_mongodb_url: "{{ml_analytics_mongodb_url | default(groups['mongo_master'][0]+':27017')}}"
+ml_analytics_mongodb_url: "{{ml_analytics_mongodb | default(groups['mongo_master'][0]+':27017')}}"
```

Comment:
give the `_url` also. The variables should be self explanatory. and you don't need the `+` sign